src/Terminus.php

```diff
@@ -138,23 +139,35 @@ public function run(InputInterface $input, OutputInterface $output)
         $status_code = $this->runner->run($input, $output, null, $this->commands);
         if (!empty($cassette) && !empty($mode)) {
             $this->stopVCR();
+        } else {
+            $this->runUpdateChecker();
```

Comment:
Also, I wonder if this should happen on startup rather than on `run`. If we do implement a shell command that would be more appropriate. Same if we have commands that call the runner to run other commands.